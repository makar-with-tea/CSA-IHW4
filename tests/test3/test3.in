3 1 1 4 test3.out