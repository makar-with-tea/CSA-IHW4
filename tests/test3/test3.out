Покупатель 3 встал в очередь в отделе 1
Покупатель 2 встал в очередь в отделе 3
Покупатель 1 встал в очередь в отделе 2
Отдел 1 обслужил покупателя 3
Отдел 2 обслужил покупателя 1
Отдел 3 обслужил покупателя 2
Покупатель 3 встал в очередь в отделе 3
Отдел 3 обслужил покупателя 3
Покупатель 3 встал в очередь в отделе 1
Отдел 1 обслужил покупателя 3
КОНЕЦ РАБОТЫ МАГАЗИНА
